### DISCUSSION TITLE
How can i implement draggable elements in Bevy ui?

### DISCUSSION THREAD

**Discussion by DraconovRyan04** [bevyengine/bevy#12871] [Q&A]
I want to implement dragable ui elements, for example, like in project zomboid. In PZ u can drag ui elements and items around game screen. Will I have to use other libraries to do this? Sorry for my poor English.

**Comment by hymm** [comment]
There's nothing built in for that at the moment. I've using bevy_mod_picking for this functionality in the past.

**Reply by DraconovRyan04** [reply]
Thx for your reply. Do u have examples in your repositories?
If there is no built-in solution, I would wait for other suggestions, such as other libraries for building the UI.
